Source/WebCore/worklets/WorkletGlobalScope.cpp

```diff
@@ -126,13 +126,19 @@ URL WorkletGlobalScope::completeURL(const String& url, ForceUTF8) const
 
 void WorkletGlobalScope::logExceptionToConsole(const String& errorMessage, const String& sourceURL, int lineNumber, int columnNumber, RefPtr<ScriptCallStack>&& stack)
 {
+    if (UNLIKELY(settingsValues().logsPageMessagesToSystemConsoleEnabled))
+        PageConsoleClient::logMessageToSystemConsole(*makeUnique<Inspector::ConsoleMessage>(MessageSource::JS, MessageType::Log, MessageLevel::Error, errorMessage, sourceURL, lineNumber, columnNumber));
```

Comment:
Why do we need a std::unique_ptr?  Can't we just make a temporary ConsoleMessage on the stack instead of allocating just to immediately discard?

Also, I'm not sure if PageConsoleClient::logMessageToSystemConsole is thread safe.  It might be.  I haven't checked.

Reply:
I'm not 100% confident, but from a quick check, the things it uses are `StringBuilder` and `WTFLogAlways()`, which I'd assume are thread safe.

Reply:
This should be safe as it is used for workers as well.